recipes/knit/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.2.2" %}
+{% set name = "knit" %}
+
+package:
+    name: {{name}}
+    version: {{version}}
+
+source:
+  fn: {{name}}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/k/{{name}}/{{name}}-{{ version }}.tar.gz
+  sha256: 3d847be729bd8f25bca7da993ecea44600ac31b90a553e03c14da575a0add004
+  
+build:
+  number: 0
+  skip: True  # [not linux]
```

Comment:
Would it make sense for this to be a `noarch: python` package?

Reply:
Actually, it probably could - but I have done absolutely no testing on not-linux. For the time being, Knit is designed to work on a YARN cluster node (edge-node), which will always be linux in practice.